src/ddmd/todt.d

```diff
@@ -719,7 +719,7 @@ private void membersToDt(AggregateDeclaration ad, DtBuilder dtb,
         {
             dtb.xoff(toVtblSymbol(concreteType), 0);  // __vptr
             offset = Target.ptrsize;
-            if (!cd.cpp)
+            if (!cd.classKind == ClassKind.cpp)
```

Comment:
Should this have been `cd.classKind != ClassKind.cpp`?

Reply:
Yes, indeed. please fix in #7378

Reply:
Done.  Thanks